wiki/Rules/Content_Voting_Process/id.md

```diff
@@ -28,7 +28,7 @@ Setelah 3 hari, ia mendapatkan kabar seputar gambar yang ia ajukan tersebut.
 
 Ternyata, hanya 52% anggota GMT dan NAT yang menyatakan bahwa gambar tersebut layak guna. 48% sisanya mengajukan keberatan.
 
-Karena suara-suara GMT dan NAT tidak mengarah pada suatu konsensus, maka (sesuai prosedur yang ada) suara-suara BN akan turut dijadikan bahan pertimbangan dalam kasus ini.
+Karena suara-suara GMT dan NAT tidak mengarah pada suatu konsensus, maka (sesuai prosedur yang ada) suara-suara BN turut dijadikan bahan pertimbangan dalam kasus ini.
```

Comment:
```suggestion
Karena suara-suara GMT dan NAT tidak mengarah pada suatu konsensus, sesuai prosedur yang ada, suara-suara BN turut dijadikan bahan pertimbangan dalam kasus ini.
```

Reply:
i think i'll just simplify this altogether into `maka suara-suara yang masuk dari para anggota BN`